### PR TITLE
add 9/14/17 releasenotes

### PR DESCRIPTION
**Pull request by ucbjrl**

### CODE REVIEW

File: release-notes-17-09-16.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+```
+
+Results in:
+```
+Error:(12, 6) value out is not a member of Chisel.Bundle
+  io.out := io.in
+Error:(12, 16) value in is not a member of Chisel.Bundle
+  io.out := io.in
+```
+*NOTE:* This seems to be constrained to compatibility mode.
```
Comment (edwardcwang) [inline release-notes-17-09-16.md]: So compatibility mode requires `-Xsource:2.11`?

File: release-notes-17-09-16.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+### Interpreter
+
+- Update sbt to 0.13.16; add Scala 2.12 support. [Firrtl-interpretere PR (#89)]:(https://github.com/freechipsproject/firrtl-interpreter/pull/89)
+
+### DSPTools
+
+Initial SNAPSHOT release of the DSP generator library.
+Please visit the [GitHub repository](https://github.com/ucb-bar/dsptools) for instructions and examples.
+*Note:* Currently, we only support Scala 2.11 versions of this library - we don't have 2.12 versions of all of its dependencies.
```
Comment (edwardcwang) [inline release-notes-17-09-16.md]: Is there an issue mixing Scala 2.12 Chisel with dsptools then?

File: release-notes-17-09-16.md (hunk 1 above)
Comment (ucbjrl) [reply release-notes-17-09-16.md]: Yes. Please see [Chisel3 issue 606](https://github.com/freechipsproject/chisel3/issues/606).

File: release-notes-17-09-16.md (hunk 2 above)
Comment (ucbjrl) [reply release-notes-17-09-16.md]: Yes. 2.11 code is incompatible with 2.12. We're still releasing 2.11 versions of the BIG4 so dsptools will continue to work. This time we're also releasing 2.12 versions of the BIG4.

Comment (edwardcwang) [reply release-notes-17-09-16.md]: Seems like we may want to write that explicitly into the notes for users?

File: release-notes-17-09-16.md (hunk 1 above)
Comment (edwardcwang) [reply release-notes-17-09-16.md]: Ditto below (would add the suggested workaround to the notes)

**Comment by ucbjrl** [comment]
Responded to reviewer 's comments.

**Pull request merged**